public/app/plugins/datasource/alertmanager/types.ts

```diff
@@ -330,7 +331,8 @@ export interface TimeInterval {
 export type MuteTimeInterval = {
   name: string;
   time_intervals: TimeInterval[];
-  provenance?: string;
+  provisioned?: boolean;
+  metadata?: ReadNamespacedTimeIntervalApiResponse['metadata'];
```

Comment:
(I should possibly make a separate type that has this property on it instead of adding to this one here, as it's now directly from the datasource)